posts/2021-10-01-ga-release-post.adoc

```diff
@@ -0,0 +1,175 @@
+* link:https://openjdk.java.net/jeps/378[Text Blocks]
+* link:https://openjdk.java.net/jeps/380[Unix-Domain Socket Channels]
+* link:https://openjdk.java.net/jeps/390[Warnings for Value-Based Classes]
+* link:https://jdk.java.net/16/release-notes#JDK-8202343[Removal of TLS 1.0 and 1.1]
+* link:https://openjdk.java.net/jeps/394[Pattern Matching for instanceof]
+* link:https://openjdk.java.net/jeps/395[Records]
+* link:https://openjdk.java.net/jeps/403[Strongly Encapsulate JDK Internals]
+* link:https://openjdk.java.net/jeps/409[Sealed Classes]
+* link:https://openjdk.java.net/jeps/411[Deprecate the Security Manager for Removal]
+* link:https://bugs.openjdk.java.net/browse/JDK-8196415[Disable SHA-1 JARs]
+* link:https://bugs.openjdk.java.net/browse/JDK-8259709[Disable SHA-1 XML Signatures]
+
+=== Phasing out the --illegal-access JVM option
+
+By far one of the the biggest impacts to Java development happened over the past couple releases. Java introduced a key play to move Java applications firmly into the modular programming world of the Java Platform Module System (JPMS).
+
+For a quick history review of JPMS, please re-visit the *Introduction of the Java Platform Module System (JPMS)* section of our link:{url-prefix}/blog/2019/02/06/java-11.html[Open Liberty and Java 11 blog].
+
+Starting in Java 16 with link:https://openjdk.java.net/jeps/396[JEP 396], strong encapsulation of the JDK internal elements took another step.  This effectively means the "kill" switch default for the JVM option `--illegal-access` was changed from `permit`, permitting module access violations, to `deny`, denying this access and instead throwing an `IllegalAccessException` when a violation occurs.
+
+Then, in Java 17 with link:https://openjdk.java.net/jeps/403[JEP 403], they took strong encapsulation further.  Now, while the JVM option `--illegal-access` is still recognized as a valid JVM argument, it is effectively hard coded to `deny`, its passed in value is ignored and the following warning message is issued.
+
+```
+OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=permit; support was removed in 17.0
+```
+
+Not to worry though.  You still have more time if your applications or dependencies are not completely ready to run in the modularized Java world.  There are some more precise JVM arguments, primarily `--add-opens` and `--add-exports`, that can help bridge the gap until you, and your applications, are fully modular ready.  These arguments allow the individual module exceptions to the JPMS rules that your applications might need.  
+
+For more information on this topic, please review the link:https://docs.oracle.com/en/java/javase/17/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B[Strong Encapsulation in the JDK] documentation or checkout one of the other helpful links in the link:<Please turn this into a link to the Java 17 resources section>[Java 17 resources] sections below.
```

Comment:
Just to make sure this does not get missed, the text `link:<Please turn this into a link to the Java 17 resources section>[Java 17 resources]` will need to be changed into a valid link.  Looks like the whole text should be changed to `<<java-17-resources, Java 17 resources>>`.

Reply:
Thanks for pointing that out!

Reply:
I unresolved this conversation because this psudo link is still showing on the staging server.